Text-Utilies.css

```diff
@@ -0,0 +1,22 @@
+@import url(components.css);
+.Text-Headings{
+    line-height: 3rem;
+}
+.texth2 .Text-Headings{
+    text-align: none;
+}
+.text
+{
+    color:var(--textgray);
+    font-size: 2rem;
+}
+.Underlined
+{
+    text-decoration: underline;
+    font-size: 2rem;
+}
+.Strike
```

Comment:
The naming convention is inconsistent.
Refer this: https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/

Reply:
Will take Care from next Time Thank you  for the Suggestion Really Appreciated